### PR TITLE
Add storybook with Panel component

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change
We need to build components directly into Data Hub FE. 

As I needed to port the Panel component from Data Hub Components to FE I needed Storybook installed so I can view it.

Note: We do need to get Jest installed in another iteration so we can unit test the components.

## Test instructions
- Run `yarn storybook`
- You should see the Panel component

## Screenshots
Screenshot 2020-04-09 at 14 20 03

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?

### CODE REVIEW

File: src/client/components/Panel/example.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Import
+```js
+import Panel from 'Panel
```
Comment (mforner13) [inline src/client/components/Panel/example.md:3]: Missing the closing ` after Panel

Comment (ian-leggett) [reply src/client/components/Panel/example.md:3]: Good spot thanks 👍

**Pull request merged**